### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nkrantz**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,62 @@
...
 
 To run Cypress tests, run:
 `yarn cypress`
 
-## Learn More
+## Start editing the prototype:
+
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
+If you'd like a more detailed tutorial, follow the steps on the landing page to familiarize yourself with the Prototyping Kit and Paste.
```
Comment (lilianaw) [inline README.md:39]: Good idea to include this in here!

File: README.md [hunk 2]
```diff
@@ -1,44 +1,62 @@
...
-## Getting Started
+After forking and cloning the repo to your local machine, run
 
-First, run the development server:
+```bash
+npm install
+# or
+yarn install
+```
+then, run the development server
```
Comment (lilianaw) [inline README.md:22]: Is there a reason there are npm and yarn instructions for this and only yarn for testing?

**Review by lilianaw — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (nkrantz) [reply README.md:22]: nope, good catch!

File: README.md [hunk 3]
```diff
@@ -1,44 +1,62 @@
 # Paste Prototyping Kit
 
-## Quick deploy
+Use this kit to create realistic prototypes using Paste and validate your designs. Read below for a guide to getting started.
+
+## Getting set up (for designers):
+
+Deploy to Codesandbox to quickly get started on your prototype
```
Comment (nkrantz) [inline README.md:7]: "Edit in the browser on Codesandbox" instead of "deploy"

**Pull request merged**